package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "oppia-bot",
+  "version": "1.0.0",
+  "description": "",
+  "author": "Apurv Bajaj <apurvabajaj007@gmail.com>",
+  "license": "ISC",
```

Comment:
Preferably it would be Apache 2.0 since that's what the main Oppia code is licensed.